torchvision/transforms/functional_tensor.py

```diff
@@ -15,6 +15,10 @@ def _assert_image_tensor(img: Tensor) -> None:
     if not _is_tensor_a_torch_image(img):
         raise TypeError("Tensor is not a torch image.")
 
+def _assert_threshold(img: Tensor, threshold:int) -> None:
```

Comment:
```suggestion
def _assert_threshold(img: Tensor, threshold: float) -> None:
```

Reply:
Thanks for the review :) It should be `float`.